lua/lualine/components/swenv.lua

```diff
@@ -1,22 +1,19 @@
 local M = require('lualine.component'):extend()
 
-local highlight = require('lualine.highlight')
-local utils = require('lualine.utils.utils')
+local default_opts = {
+  icon = "",
+  color = { fg = "#8fb55e" },
```

Comment:
[nit] maybe this color instead which is more like the python logo? but not so important

```suggestion
  color = { fg = "#FFD43B" },
```

Reply:
sure, that works.